kapitan/targets.py

```diff
@@ -100,21 +99,21 @@ def compile_targets(inventory_path, search_paths, output_path, parallel, targets
 
         # Save inventory and folders cache
         save_inv_cache(compile_path, targets)
-
+        pool.close()
     except Exception as e:
         # if compile worker fails, terminate immediately
         pool.terminate()
-        pool.join()
```

Comment:
Not joining here might obfuscate the "Compile pool terminated" with other output.

Reply:
tested and stacktraces appear as they should.